climateconnect/climateconnect
# Fix: Search own name in inbox results

## Description
Closes #1061 

Added a .exclude() to the QuerySet which should trigger when the result uses the current user as a possible result.
Included changes from #1082 and suggestions to speed up search query.
In inbox.js removed loading statements that would cause the UI to snap to top every page. 

## Test plan
1. Sign in
2. Navigate to the inbox
3. Click "find a chat"
4. Search your own name
5. You should not see every possible open chat you have
## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
